tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -449,7 +449,7 @@ def test_various_options(self, mock_get_repositories):
             "genAIConsent": True,
             "issueAlertsThreadFlag": False,
             "metricAlertsThreadFlag": False,
-            "metricsActivatePercentiles": True,
+            "metricsActivatePercentiles": False,
```

Comment:
`metricsActivatePercentiles` are now by default true, so this test checks if it is possible to update them to non-default value (to false) and if the audit log will contain record of that update. Audit log entry check can be found on the line 521 in this file